docs/xdr/features/investigate/query_builder.md

```diff
@@ -0,0 +1,128 @@
+# Query Builder (beta)
+
+Start exploring your data with the Query Builder. Hunt for threats, get analytics, create reports. Query Builder is a simple form which allows you to build queries to explore your data. 
+Using a SQL syntax, you can now aggregate your data to extract new insights. Query Builder also offer the possibility to visualize data through different kind of charts.
+
+At the moment, only the `Events` table is available but more tables are coming: Alerts, Cases, Assets, CTI and more.
+
+![query builder](/assets/operation_center/events/query-builder.png){: style="max-width:100%"}
+
+## Build your query
+
+Select a table to query in the `From` clause and use `Add clause` button to refine your query.
+
+![clauses](/assets/operation_center/events/qb-clauses.png){: style="max-width:100%"}
+
+The Query Builder comes with standard SQL clauses.
+
+| Clause | Description |
+| --- | --- |
+| SELECT | Choose columns you want to display |
+| AGGREGATION | Aggregate rows with `average` / `count` / `min` / `max` / `sum` / `unique count` |
+| WHERE | Define query conditions |
+| GROUP BY | Group rows by identical data |
+| ORDER BY | Order rows by a specific value |
+| LIMIT | Limit the number of rows in results |
+
+### Aggregation
+
+Create an aggregation by using the `Aggregate` clause. Select a type of aggregation, a column and a label for your aggregation.
+
+![aggregation](/assets/operation_center/events/qb-aggregation.png){: style="max-width:100%"}
+
+The following aggregation types are available.
+
+| Aggregation type | Description |
+| --- | --- |
+| Average | Average of a set |
+| Count | Count the number of rows in a set |
+| Min | Find the minimum value in a set |
+| Max | Find the maximum value in a set |
+| Sum | Sum of a set|
+| Unique count | Count unique values of a set |
+
+### Operators
+
+Use the following operators to define your conditions in the `WHERE` clause.
+
+| Operator | Description |
+| --- | --- |
+| is | Equals |
+| is not | Not equals |
+| starts with | Starts with the specified value. `starts with` is case-sensitive |
+| does not start with | Does not start with the specified value. `does not start with` is case-sensitive |
+| ends with | Ends with the specified value. `ends with` is case-sensitive |
+| does not end with | Does not end with the specified value. `does not end with` is case-sensitive |
+| contains | Contains the specified value. `contains` is case-sensitive |
+| does not contain | Does not contain the specified value. `does not contain` is case-sensitive |
+| exists | Has a value |
+| does not exists | Does not have a value |
+| < | Strictly less than |
+| <= | Less than or equal to |
+| > | Strictly more than |
+| >= | More than or equal to |
+
+!!! note
+    Operators `starts with`, `does not start with`, `ends with`, `does not end with`, `contains`, `does not contain` are case-sensitive
```

Comment:
All operators are case sensitive (including `is` and `is not`). I feel like listing them may suggest that some of them are not.

Reply:
Done